docs/plugins.md

```diff
@@ -76,7 +76,7 @@ buildscript {
     maven { url "https://repo1.maven.org/maven2" }
   }
   dependencies {
-    classpath "org.openapitools:openapi-generator-gradle-plugin:3.3.4"
+    classpath "org.openapitools:openapi-generator-gradle-plugin:4.3.1"
```

Comment:
What about updating it to 5.0.0 - the latest stable release?

Reply:
Have updated to 5.0.0 for both maven and gradle